### ISSUE TITLE
tensorflow r1.0 does not report training speed in the log any more

### ISSUE THREAD

**Issue by jinghuangzhu** [tensorflow/tensorflow#8145]
I used tensorflow r0.10 before with contrib.learn.
When I set 
tf.logging.set_verbosity(tf.logging.INFO)

after imports, I would get the following info from the training log:

INFO:tensorflow:Results after 10 steps (0.185 sec/batch): loss = 0.0644001, auc = 0.73555, accuracy/threshold_0.500000_mean = 0.988108

But with tensorflow r1.0, no such info in the training log any more.
Why is it removed? It is such a useful information.
or do I need to set something else in the script?

Thanks for help.

**Comment by prb12** [comment]
That log message appears to still be in the source code here:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/learn/python/learn/graph_actions.py#L786

However, it is predicated on the `log_every_steps` parameter.  Perhaps this is being configured differently by default.

In any event, this code appears to be deprecated now, so I wouldn't suggest depending on the existence of this particular log message.  @ispirmustafa may know more.

I'm closing this issue, since it's unlikely to be reverted.
